appendix-sample-contract.md

```diff
@@ -28,6 +28,13 @@ guidance as you strike your own contracts with vendors.  We include
 this template because of its more in-depth security requirements,
 which, again, may or may not be relevant to your situation.
 
+While this sample contract language is provided under the Creative Commons
```

Comment:
Does this mean the TODO immediately below is now resolved?

Reply:
If no one else has better sample contract language, the TODO can be resolved. When this is published on main, I will reconnect with the author and share this language with him to be sure he's ok with it. It's exactly what he asked and should be fine, this is more a courtesy.